tracing/README.adoc

```diff
@@ -1,66 +0,0 @@
-//
-// Copyright (c) 2022 Contributors to the Eclipse Foundation
-//
-// See the NOTICE file(s) distributed with this work for additional
-// information regarding copyright ownership.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-
-= MicroProfile Telemetry Tracing
-This specification is to adopt https://opentelemetry.io/[OpenTelemetry] from CNCF to allow MicroProfile applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically being traced if configured.
```

Comment:
The removal of this README made me notice that the `README.adoc` in the root of the repository should be updated because it has a "Structure of this Specification" section which includes the text

> This specification potentially adopts all three main parts when they are stable in OpenTelemetry. If this is the case, **they will appear as submodules in this specification**. For instance, you can find details related to tracing from the directory of `tracing`.